src/openms/include/OpenMS/MATH/STATISTICS/PosteriorErrorProbabilityModel.h

```diff
@@ -168,6 +193,9 @@ namespace OpenMS
       void tryGnuplot(const String& gp_file);
 
 private:
+      /// transform different score types to a range and score orientation that the model can handle (engine string is assumed in upper-case)
```

Comment:
what Engines are allowed?
Use an enum?

Reply:
I will put it in the documentation. But I agree that we could use less String constants and use more enums instead